bin/rose

```diff
@@ -44,7 +44,7 @@ rose_init
 get_alias() {
     local NAME=$1
     local ALIAS=$(sed '/^#/d' $ROSE_HOME_BIN/$ROSE_NS-$NAME || true)
-    if [[ $(wc -l <<<"$ALIAS") == 1 ]] \
+    if [[ $(wc -l <<<"$ALIAS") ]] \
```

Comment:
This bit of logic is getting way too complicated. I am sure there is a way to do a single `grep` or something. Not sure if this is in the scope of this change or not, however.

Reply:
Ah thats a rebase error, i'll remove this diff.